tests/ConsensusReporter_test.py

```diff
@@ -0,0 +1,57 @@
+import io
+import os
+import shutil
+from datetime import datetime
+from unittest import TestCase
+
+from consensus.ConsensusReporter import ConsensusReporter
+
+
+class TestConsensusReporter(TestCase):
```

Comment:
Test fails on my windows machine with error:
```
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'test_data\\tmp\\'
```
I think temporary directories should be made using https://docs.python.org/3/library/tempfile.html.

Once this error has been fixed I expect the test to fail because the line separator in the expected data does not match my os line separator (see comments in consensus/ConsensusReporter.py).

Reply:
According to @marikaris the test can't be run from PyCharm but needs to be run from the command-line (see README.md for instructions).